get_security_annotations.py

```diff
@@ -100,20 +101,13 @@ def __has_current_page_items(self):
         return self.current_page_items_pointer < len(self.current_page_items)
 
 
-def get_asset(asset_id):
-    response = requests.get(API_HOST_NAME + build_asset_url(asset_id),
-                            headers={
-                                "Authorization": get_authorization(
-                                    SECUREX_VISIBILITY_HOST_NAME,
-                                    SECUREX_CLIENT_ID,
-                                    SECUREX_CLIENT_PASSWORD
-                                ),
-                                "Accept": "application/json"
-                            })
-    return response.json()
+def get_object(object_url):
+    global object_cache
```

Comment:
it might be better to wrap all the functions in some kind of object, than you wouldn't need mutable global state

Reply:
Removed this code when rewriting it to bulk.